styles/pypi/catppuccin.user.css

```diff
@@ -135,6 +135,10 @@
       background-image: linear-gradient(90deg, @surface0, @surface0) !important;
     }
 
+    .project-description blockquote {
+      color: @text !important;
```

Comment:
i think this should be subtext0 as in the default styles its a more muted color than the main text